docs/frontend/testing/writing-playwright-tests-for-milmove.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 9
+---
+
+# Writing Playwright Tests for MilMove
+
+This guide covers how MilMove writes tests for Playwright. You should
+probably familarize yourself with the [playwright docs on writing
+tests](https://playwright.dev/docs/writing-tests). Some of the best
+practices below merely reinforces information from that doc.
+
+You can almost certainly use some of the knowledge from [Writing Tests using React Testing Library and Jest](writing-tests-using-react-testing-library-and-jest.md).
+
+## Best Practices / Things to Know
+
+### No seed data
+Unlike our old cypress testing strategy of running a command to load
+seed data with hard coded ids that tests can reference, with
+playwright if a test needs test data in the database, it should create
+it using the testharness in the support api. See below for more
+details.
+
+You should be able to run any playwright test after a `make
+db_dev_reset db_dev_migrate` or after `make db_dev_truncate`.
+
+### Always (always!) ensure your test runs independently
+When adding a test, you should ensure it can run on its own without
+any other test. Never have one test depend on another test having run.
+That's how we get flaky tests!
+
+### Test data growth can (eventually) cause test failures
+The majority of the time, playwright tests will be run on an empty
+database. If you are developing locally and run the tests many time,
+each test will need create its own data. That may eventually cause some
```

Comment:
**question:** What do you mean by "that"? I'm a bit confused here with the sentence "That may eventually...".